Source/Scene/Scene.js

```diff
@@ -1867,6 +1867,9 @@ define([
             }
 
             clearDepth.execute(context, passState);
+            if (context.stencilBuffer) {
+                scene._stencilClearCommand.execute(context, passState);
+            }
```

Comment:
This is why this command is added here. Without it it breaks between multiple frustums because the stencil buffer isn't cleared, but here we just reset the stencil state between frustums.